gpbft/types.go

```diff
@@ -1,8 +1,9 @@
 package gpbft
 
 import (
-	"github.com/ipfs/go-datastore"
 	"math/big"
+
+	"github.com/ipfs/go-datastore"
```

Comment:
Drive-by: this datastore and network stuff doesn't really belong in the core gpbft. Could we move these methods out to pure functions closer to where they are called? The datastore key in particular is at the host's discretion.